.circleci/config.yml

```diff
@@ -10,70 +11,73 @@ jobs:
       tag: "17.0"
     steps:
       - checkout
-      - run: mkdir -p ~/build-artifacts
+      - run: mkdir -p ./build-artifacts
```

Comment:
Work with build artifacts in the working directory instead of HOME so that the make targets to generate and use them work equally well on a laptop and in CI.